src/bundle/DependencyInjection/SchebTwoFactorExtension.php

```diff
@@ -234,6 +235,28 @@ private function configureTotpAuthenticationProvider(ContainerBuilder $container
         $container->setAlias('scheb_two_factor.security.totp.form_renderer', $config['totp']['form_renderer']);
     }
 
+    /**
+     * @param array<string,mixed> $config
+     */
+    private function configureCodeReuseCache(ContainerBuilder $container, array $config): void
+    {
+        if (($config['code_reuse_cache'] ?? null) === null) {
+            $config['code_reuse_cache'] = '';
+        }
+
+        if (($config['code_reuse_cache_duration'] ?? null) !== null) {
+            $container->setParameter('scheb_two_factor.code_reuse_cache_duration', (int) $config['code_reuse_cache_duration']);
+        }
```

Comment:
To my understanding `$config['code_reuse_cache_duration']` will always be an integer. When the config value isn't set, the default integer value will be provided. So I believe the parameter can always be set without any extra checks.

```suggestion
        $container->setParameter('scheb_two_factor.code_reuse_cache_duration', $config['code_reuse_cache_duration']);
```